sources/europe/md/Ortophoto_2016_RM_Reserve-Server.geojson

```diff
@@ -0,0 +1,1741 @@
+{
+    "type": "Feature",
+    "properties": {
+        "description": "2016 Ortophotos, Moldova, Scale 1:5000, 20cm, true colours, covers practically the entire territory of the Republic of Moldova, except some areas from the east part of Moldova. [Geoportal.md server]",
+        "id": "Geoportal-Ortofoto_2016_RM",
+        "name": "Moldova Ortophoto (2016) / Geoportal.md Reserve Server",
+        "type": "wmts",
```

Comment:
I assume this should by `"type": "wms"`

Reply:
Ah! Yeah, that's true, here should to be wms.

Reply:
Thanks for the help!